### PR TITLE
Aaf 103 pairing wishlist

### PR DESCRIPTION
**Pull request by rainrivas**
## Added
- Route for pairing families to donors (or orgs) to `/api/pairing/:familyId`
- Pairing Controller placeholder file `server/controller/pairingController.js`
- Pairing tests to `server/routes/__tests__/PairingApi.test.js`, but they're all skipped
- New constants have been added to PairingApi test
## Removed
- Route `api/pairing/` has been removed (see new routes"
- All references to old pairing endpoint,`/api/pairing`, have been taken out of PairingTest and api routes (`server/routes/api.js`)
## Changed
- Route to find families based on budget has changed from `/api/pairing` to `/api/pairing/budget`
- Tests now call new constants that were added to file
- Base pairing tests are now called calling `BUDGET_PAIRING_ENDPOINT`

### CODE REVIEW

File: server/routes/__tests__/PairingApi.test.js [hunk 1]
```diff
@@ -65,41 +67,140 @@ describe('Test /api/pairing', () => {
...
+  });
+
+  describe('Other error messages', () => {
+    it.skip('Should return 500 if there are errors grabbing data from db', () => {
+      // Make this work somehow...
+    });
+  });
+});
+
+/******************************
```
Comment (rainrivas) [inline server/routes/__tests__/PairingApi.test.js:106]: I should add a TODO for the skips...

File: server/controllers/paringController.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
```
Comment (phuchle) [inline server/controllers/paringController.js]: Spelling on file name.

**Review by phuchle — COMMENTED** [review]
Is the plan to get these changes in and do another ticket for the unfinished work?

**Comment by rainrivas** [comment]
@phuchle 
> Is the plan to get these changes in and do another ticket for the unfinished work?

Yes. Going to get everything I've done up to this point into development, and open the rest of the stuff up as TODO and then handle it in the TODO tickets. I'll link them to 103, so my plan was something link
"AAF-103/TODO-##-populate-wishlist" or something like that.

**Comment by rainrivas** [comment]
@5-gwoap/adopt-a-family I have made corrections

**Review by phuchle — DISMISSED** [review]
*points to forehead* tests can't fail if there are no tests

**Comment by phuchle** [comment]
@rainrivas Can you pick out what messages you want to be squashed here?  Can you also make sure to note that in this commit that the tests weren't included?

**Comment by rainrivas** [comment]
@5-gwoap/adopt-a-family I've updated the description with what i want to put in the squash. Also pushed up some last minute change. Let me know if I should build the app and push up those things as well. Otherwise, we should be good once this job is done executing.

**Review by phuchle — APPROVED** [review]

**Review by jwu910 — APPROVED** [review]
LGTM

**Pull request merged**